.github/workflows/build-with-bal-test-graalvm.yml

```diff
@@ -0,0 +1,19 @@
+name: GraalVM Check
+
+on:
+  schedule:
+    - cron: "30 18 * * *"
```

Comment:
Let's change the time here. Most of the libraries using the same cron expression, resulting every build starting to run at the same time. This might cause to exceed the GitHub workers limit.